### PR TITLE
Docs for proposed new Sidebar

### PR DESCRIPTION
**Pull request by twMat**
I'd prefer headline size "!!" rather than "!" but I'm following formatting found in some other tiddler.

Obviously, improve the doc in any way suitable.

### CODE REVIEW

File: editions/tw5.com/tiddlers/howtos/Modifying the Sidebar [hunk 1]
```diff
@@ -0,0 +1,27 @@
+created: 20170204200200000
+modified: 20170204200200000
+tags: [[Customize TiddlyWiki]] Learning 
+title: Modifying the Sidebar
+type: text/vnd.tiddlywiki
+
+The [[Sidebar|$:/core/ui/PageTemplate/sidebar]] consists of several sections that are really [[transcluded|Transclusion]] tiddlers. You can easily add, remove and reorder the sections.
```
Comment (danielo515) [inline editions/tw5.com/tiddlers/howtos/Modifying the Sidebar]: >remove and reorder the sections

What about "such sections" ? Not sure about it, but how does it sound to you?

**Comment by twMat** [comment]
I agree that "Tag it with" is better and have made the change. Not sure about "such sections" though - we'll let the natives decide. Thanks for input @danielo515 
I also changed the titles mentioned under "See also" into only lower case initials, which is the intended titles format policy from @Jermolene 

( @danielo515 - so no misunderstanding; what I really need more help with is submitting the actual code from the [tiddlyspot](http://newsidebar.tiddlyspot.com). Apparently it requires TW on node.js )

**Comment by pmario** [comment]
@twMat ... IMO this one can be closed, since it describes a different PR, that wasn't merged.

**Comment by Jermolene** [comment]
@twMat do you think it's worth updating this to reflect the implementation in #2758?

**Comment by pmario** [comment]
@twMat ... The PR repo seems to be removed already. Do you want to keep the PR open?

**Comment by twMat** [comment]
thanx @pmario , closing

**Pull request closed without merging**